packages/functions-utils/src/main.js

```diff
@@ -45,8 +40,10 @@ const getSrcGlob = async function (src, srcBasename) {
   return srcBasename
 }
 
+const isEmptyArray = (input) => Array.isArray(input) && input.length === 0
```

Comment:
Wouldn't `functionsSrc` always be `undefined` or an array, therefore, could be directly `functionsSrc.length === 0`?

Reply:
Done in https://github.com/netlify/build/pull/3630/commits/99321115d53ad4824244177f7bd8baef68cbe520.